ui/page/wallets/wallet_page.go

```diff
@@ -1021,8 +1022,15 @@ func (pg *WalletPage) layoutAddWalletMenu(gtx layout.Context) layout.Dimensions
 		return pg.Theme.Shadow().Layout(gtx, func(gtx C) D {
 			return pg.optionsMenuCard.Layout(gtx, func(gtx C) D {
 				return (&layout.List{Axis: layout.Vertical}).Layout(gtx, len(pg.addWalletMenu), func(gtx C, i int) D {
-					return pg.addWalletMenu[i].button.Layout(gtx, func(gtx C) D {
-						return layout.UniformInset(unit.Dp(10)).Layout(gtx, pg.Theme.Body2(pg.addWalletMenu[i].text).Layout)
+					card := pg.Theme.Card()
+					card.Radius = decredmaterial.Radius(0)
+					return card.HoverableLayout(gtx, pg.addWalletMenu[i].button, func(gtx C) D {
+						return pg.addWalletMenu[i].button.Layout(gtx, func(gtx C) D {
+							return layout.UniformInset(values.MarginPadding10).Layout(gtx, func(gtx C) D {
+								gtx.Constraints.Min.X = gtx.Constraints.Max.X
+								return pg.Theme.Body2(pg.addWalletMenu[i].text).Layout(gtx)
+							})
+						})
```

Comment:
Looking again, you really dont need the hoverable card 

```suggestion
				return pg.addWalletMenu[i].button.Layout(gtx, func(gtx C) D {
					return layout.UniformInset(values.MarginPadding10).Layout(gtx, func(gtx C) D {
						gtx.Constraints.Min.X = gtx.Constraints.Max.X
						return pg.Theme.Body2(pg.addWalletMenu[i].text).Layout(gtx)
					})
```